### PR TITLE
fix(rac): update embedded-code.js. Avoid `executeScript()` twice

### PR DESCRIPTION
**Pull request by nickhsine**
### Issue
slack msg: https://twreporter.slack.com/archives/C09N6TNN7/p1717123213877979

### Notice
After upgrading react to v18, `EmbeddedCode` component becomes abnormal. One case is that `EmbeddedCode` will `executeScript()` twice; therefore, it will generate duplicate contents.

This patch is a workaround to avoid `executeScript()` twice.

### CODE REVIEW

File: packages/react-article-components/src/components/body/embedded-code.js [hunk 1]
```diff
@@ -221,7 +234,7 @@ const WayPointWrapper = props => {
       onEnter={onEnter}
       onLeave={onLeave}
       fireOnRapidScroll={false}
-      topOffset={5}
+      topOffset="-150%"
```
Comment (Aylie-Chou) [inline packages/react-article-components/src/components/body/embedded-code.js:237]: Question: why do we change the `topOffset` value here ?

Comment (nickhsine) [reply packages/react-article-components/src/components/body/embedded-code.js:237]: Oops... This change is not intended.
Thank you for pointing out.

I need to add `bottomOffset="150%"` instead.
The reason to add `bottomOffset` is that I want to execute scripts earlier. 
Therefore, the ScrollableVideo will be loaded before users scroll into the viewport.

Comment (nickhsine) [reply packages/react-article-components/src/components/body/embedded-code.js]: https://github.com/twreporter/twreporter-npm-packages/pull/548/commits/0813b6b5f12b22f6ac8831c13a59f678d400e34f fixes the review comment.

Comment (Aylie-Chou) [reply packages/react-article-components/src/components/body/embedded-code.js]: Thanks for the detail explanation.
Just want to confirm that do we need to fire all scripts earlier or just `story-telling` scripts ?

Comment (nickhsine) [reply packages/react-article-components/src/components/body/embedded-code.js]: For better user experience, I think we should execute all scripts earlier; therefore, users won't see the content shifting. 
But, I am not sure what side effect would be for that change (add `bottomOffset`).
How do you think?

**Review by Aylie-Chou — APPROVED** [review]
LGTM 👍

**Pull request merged**